Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -259,6 +330,72 @@ def virtual_service_create_command(client: PolicyComputeEngine, args: Dict[str,
     )
 
 
+def service_binding_create_command(client: PolicyComputeEngine, args: dict[str, Any]) -> CommandResults:
+    """Create a service binding.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        CommandResult object
+    """
+    workloads = args.get("workloads")
+    virtual_service = args.get("virtual_service")
```

Comment:
We should use `convert_draft_href_to_active` here as an active VS HREF is required for the service binding POST. It would also be good to implement a check here to see if an active version of the virtual service exists, e.g.

```
virtual_service = convert_draft_href_to_active(args.get("virtual_service"))
...  # validation
try:
    active_virtual_service = pce.virtual_services.get_by_reference(virtual_service)
except IllumioException as e:
    ...  # no active record for virtual service with HREF {href}
```